requirements_all.txt

```diff
@@ -64,6 +64,9 @@ cherrypy==8.1.0
 coinmarketcap==2.0.1
 
 # homeassistant.scripts.check_config
+colorama<=1
```

Comment:
@robbiet480 
I am.... As a check I did a clone of the original repo (balloob's) and did a `python script/gen_requirements_all.py`. It also adds the colorama entry. So there is something going wrong...